packages/lib-classifier/src/store/FeedbackStore.js

```diff
@@ -0,0 +1,109 @@
+import { autorun } from 'mobx'
+import { addDisposer, getRoot, types } from 'mobx-state-tree'
+import _ from 'lodash'
+
+import * as helpers from '../features/feedback/shared/helpers'
+import strategies from '../features/feedback/shared/strategies'
+
+const FeedbackStore = types
+  .model('FeedbackStore', {
+    active: types.optional(types.boolean, false),
+    rules: types.frozen({}),
+    modal: false
+  })
+  .views(self => ({
+    get currentRules () {
+      if (self.active) {
+        return self.rules
+      }
+      return null
+    },
+    get currentRulesValues () {
+      const values = _.flatten(_.values(self.rules))
+      if (values.length) {
+        return values
+      }
+      return null
+    },
+    get feedbackMessages () {
+      const messages = _.chain(self.currentRulesValues)
```

Comment:
If `rules` and `messages` are both arrays then we could get rid of lodash here and write this more simply as a map and filter. Something like:
```javascript
const messages = rules.map( rule => ruleMapFunction ).filter(Boolean)
```
where `ruleMapFunction` returns `item.successMessage` or `item.failureMessage` or `false`, though I'd prefer if the return type was always a string, for consistency.